### ISSUE TITLE
Nunit Agent leads to very high Power Usage and high CPU

### ISSUE THREAD

**Issue by NadeemBader** [nunit/nunit#3774]
Hi
I noticed that the Nunit Agent leads to very high Power Usage and high CPU.
Any idea?
Best Regards,
Nadeem Bader

**Comment by mikkelbu** [comment]
No. And I don't think I've seen reports on this before, so I guess it is in your setup, but from the question it is impossible to tell. Transferring this to the nunit-console repo, but I doubt anyone can answer this unless more information is provided. Have you tried to profile the processes when you run tests?
